cmd/gen-go-sample/main.go

```diff
@@ -353,14 +353,38 @@ func (g *generator) genSample(sampConf schema_v1p2.Sample, methConf GAPICMethod)
 
 	p := g.pt.Printf
 
-	// Region tag, function signature and new client
+	// Region tag
 	argStr, err := argListStr(initInfo, g)
 	if err != nil {
 		return err
 	}
 	p("")
 	p("// [START %s]", sampConf.RegionTag)
 	p("")
+
+  // generate comments above the sample function
+  var emptyArr []string = nil
```

Comment:
I was more thinking you'd just supply `nil` to the function call `writeComment("", nil, g)` because `writeComment` will know how to handle it.

You could also wrap `writeComment` with a `writeCommentLine(gen *generator) error` method

```go
// writeCommentLine adds a commented line break to a comment block
func writeCommentLine(gen *generator) error {
  return writeComment("", nil, gen)
}
```


The latter suggestion might actually read better IMO. And wrapping the "generic" method to do something specific in its scope is totally reasonable.

WDYT?

Reply:
Done.